Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/TabbedPage.cs

```diff
@@ -65,5 +65,31 @@ public static int OffscreenPageLimit(this IPlatformElementConfiguration<Android,
 			SetOffscreenPageLimit(config.Element, value);
 			return config;
 		}
+
+		public static readonly BindableProperty TabsPlacementProperty = 
+			BindableProperty.Create("TabsPlacement", typeof(TabsPlacement), 
+			typeof(TabbedPage), TabsPlacement.Top);
+
+
+		public static TabsPlacement GetTabsPlacement(BindableObject element)
+		{
+			return (TabsPlacement)element.GetValue(TabsPlacementProperty);
+		}
+
+		public static void SetTabsPlacement(BindableObject element, TabsPlacement value)
+		{
+			element.SetValue(TabsPlacementProperty, value);
```

Comment:
Help here: as this is an enum, my understanding is that this will always have a value (Default), so I probably should not throw on the first value change. Where should I track the value change?

Reply:
If you rebase to master you'll see a property on BindableObject now that lets you test if something has been changed from the default
https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Core/BindableObject.cs#L55

Which should be sufficient

My other thought would be to add a Changing event to the bindable property. Then inside the Tab Renderer once the layout call happens it  could set a local value indicating this is how it is on the layout. The changing event could then check that... But that's probably overkill and just checking IsSet should be enough

Reply:
I'll try the rebase. I'm not the best at using GIT, let's see what happens...

Reply:
Should be done. Please check if the exception message makes sense.

Reply:
@mikescandy  Hey mike, looks like your pretty much done with this, when can we mortals expect this as a nuget-package? I suppose its going into pre-release.

Reply:
Error message seems to work just remember to set Default as the defaultValue
```
		public static readonly BindableProperty TabsPlacementProperty = 
			BindableProperty.Create("TabsPlacement", typeof(TabsPlacement), 
			typeof(TabbedPage), TabsPlacement.Top);
```

And then at this point Default and Top will just cause the same behavior